zingolib/src/wallet.rs

```diff
@@ -1708,10 +1778,24 @@ impl LightWallet {
         }
     }
 
+    pub async fn get_shieldable_sapling_notes(&self) -> Vec<SpendableSaplingNote> {
+        self.get_all_domain_specific_notes::<SaplingDomain<zingoconfig::ChainType>>()
+            .await
+            .into_iter()
+            .filter(|note| note.note.value().inner() > u64::from(MARGINAL_FEE))
+            .collect::<Vec<_>>()
+    }
+    pub async fn get_shieldable_transparent_utxos(&self) -> Vec<ReceivedTransparentOutput> {
+        self.get_unspent_utxos()
+            .await
+            .into_iter()
+            .filter(|utxo| utxo.value > u64::from(MARGINAL_FEE))
+            .collect::<Vec<_>>()
+    }
     pub async fn tbalance(&self, addr: Option<String>) -> Option<u64> {
         if self.wallet_capability().transparent.can_view() {
             Some(
-                self.get_utxos()
+                self.get_unspent_utxos()
```

Comment:
`unspent` implies there can be a `spent UTXO` aka a `spent unspent transcation output` ?